tests/Doctrine/Tests/DBAL/Schema/SchemaDiffTest.php

```diff
@@ -48,6 +48,11 @@ private function createPlatform(bool $unsafe)
             ->method('getCreateSchemaSQL')
             ->with('foo_ns')
             ->will($this->returnValue('create_schema'));
+
+        $platform->expects($this->any())
+            ->method('supportsCreateDropForeignKeyConstraints')
+            ->will($this->returnValue(true));
```

Comment:
We can omit `expects()` call when using `->expects($this->any()`.
Also, `->willReturnValue(true)` is much simpler too:

```php
$platform->method('supportsCreateDropForeignKeyConstraints')
         ->willReturnValue(true);
```

Reply:
This fails `Error: Call to undefined method PHPUnit\Framework\MockObject\Builder\InvocationMocker::willReturnValue()
`

Reply:
My bad, it's `willReturn()` :man_facepalming: